downloads/api/oai/shipcloud_v1_oai3.json

```diff
@@ -1157,6 +1157,148 @@
         }
       }
     },
+    "/pickup_dropoff_locations": {
+      "parameters": [],
+      "get": {
+        "description": "Search pickup dropoff locations by address or geographical coordinates.",
+        "responses": {
+          "200": {
+            "description": "A successful response with a list of pickup dropoff locations.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "pickup_dropoff_locations": {
+                      "type": "array",
+                      "items": {
+                        "$ref": "#/components/schemas/pickup_dropoff_location"
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "400": {
+            "$ref": "#/components/responses/400"
+          },
+          "401": {
+            "$ref": "#/components/responses/401"
+          },
+          "402": {
+            "$ref": "#/components/responses/402"
+          },
+          "403": {
+            "$ref": "#/components/responses/403"
+          },
+          "404": {
+            "$ref": "#/components/responses/404"
+          },
+          "500": {
+            "$ref": "#/components/responses/500"
+          }
```

Comment:
Yeah, we shouldn't just copy them, you're right. We answer with 404 if the plan feature is disabled, though. 403 is `:forbidden` in rails, we seem to use it for shipments & pickup requests - this could have been another valid option instead of 404, but we should choose only one here. Maybe we need to discuss this topic once again. 
402 can also happen: each plan has an API call counter (maybe some have unlimited requests, but most plans have a limit), if it is exceeded, we will answer with 402 = `:payment_required`

Reply:
I removed the 404 response, since we now respond with 403